### PR TITLE
Add Kirigami UI

### PR DESCRIPTION
**Pull request by LNJ2**
Just a WIP PR, so everyone can see what I am doing.

### CODE REVIEW

**Comment by LNJ2** [comment]
2016-11-28-192116_swaygrab

**Comment by LNJ2** [comment]
All qml code is now in Kirigami. Still something missing?

**Comment by LNJ2** [comment]
2016-11-29-225534_swaygrab
2016-11-29-225750_swaygrab

File: src/qml/LoginPage.qml [hunk 1]
```diff
@@ -1,50 +1,65 @@
...
+Kirigami.ScrollablePage {
+	id: page
+	title: "Login"
+	Layout.fillWidth: true
+	implicitWidth: applicationWindow().width
+
+	ColumnLayout {
+		width: parent.width
+
+		Controls.Label {
```
Comment (MRZA-MRZA) [inline src/qml/LoginPage.qml]: Login page looks a bit ugly. Please, put Label and TextField into a container like RowLayout.
Like this:
```
		RowLayout {
			Controls.Label {
				text: "Your Password:"
			}
			Controls.TextField {
				id: passField
				placeholderText: "Password"
				echoMode: TextInput.Password
				Layout.minimumWidth: parent.width * 1.5
				Layout.alignment: Qt.AlignHCenter
			}
		}
```

**Comment by LNJ2** [comment]
Please pay attention that the commit description  contain all the small commit descriptions (the second field)

**Comment by MRZA-MRZA** [comment]
Looks okayish now.

**Comment by LNJ2** [comment]
@MRZA-MRZA The (automatically) commit description is not good so:
```
Add Kirigami UI (#33)

* Add Kirigami UI [1/?]

* Fix roster is not opened

* Add kirigami-roster page

* Remove connection label; Button will show 'Connecting...'

I set the QtQuick version to 2.0 in all files, because we only
need that version (at least at the moment).

* Remove unnecessay imports

* Use GridLayout
```

PLEASE remove all unnecessary/illogical, before merging a PR.

A better one:
```
Add Kirigami UI (#33)

I set the QtQuick version to 2.0 in all files, because we only
need that version (at least at the moment).

The connect button will show 'Connecting...' (italic), when
pressed now.
```

**Pull request merged**